spec/rails/doc/openapi.json

```diff
@@ -341,6 +341,48 @@
         }
       }
     },
+    "/optional_params(/{id})": {
```

Comment:
@mercedesb Could you check OpenAPI documentation to verify `"/optional_params(/{id})"` is valid format of optional path parameter?
I've looked https://swagger.io/docs/specification/paths-and-operations/
 but could not find

Reply:
Ahhh good call, it does not. [Here's an open issue on the OpenAPI Specification repo where people are asking for it](https://github.com/OAI/OpenAPI-Specification/issues/93). I'll close this. Thanks for asking this question! I didn't think to check this.